src/tests/inmemory-modeler/inmemory-modeler.cpp

```diff
@@ -183,7 +183,7 @@ void LinearProblemBuildingFixture::createModelWithSystemModelParameter(
     std::vector<Constraint> constraints;
     for (const auto& [id, expression]: constraintsData)
     {
-        constraints.push_back(std::move(Constraint(id, createExpression(expression))));
+        constraints.push_back(Constraint(id, createExpression(expression)));
```

Comment:
Even better : in-place construction
```suggestion
        constraints.emplace_back(id, createExpression(expression));
```
This avoids creating a temporary `Constraint` object, copying to the vector and deleting right after.

See [example](https://gist.github.com/flomnes/946cfeb4bd1dca0e467a8304d895e340)